### DISCUSSION TITLE
Event Data

### DISCUSSION THREAD

**Discussion by Snipppppy** [Tharylia/Blish-HUD-Modules#153] [General]
Hey o/ to the Event Table team, Hey @Tharylia 

I'm Working on a small, web based, meta event timer.
[https://github.com/Snipppppy/GW2-Simple-Timer](https://github.com/Snipppppy/GW2-Simple-Timer)

I was wandering where you got the EventData from.
I found [https://api.guildwars2.com/v1/event_details.json](https://api.guildwars2.com/v1/event_details.json) but since there are all kinds of crap events in there and also no timestaps or waypints, it's kind of useless - no?
Did you build your own database - and if so, where can i steal it? 😂

ty

**Comment by Tharylia** [comment]
Hey 😀

The link you posted from the v1 api is indeed not that helpful for timers as it contains all dynamic events (up to PoF).
The other endpoint that was showing the time as well got deactivated with the megaserver addition. A v2 alternative was planned but never made it.

This is why i have my own definition file for all timer based events.
You can find that [here](https://files.estreya.de/blish-hud/event-table/v1/events.json).

Feel free to copy as much as you like.
Just don't depend on the file as I can't guarantee it won't have breaking changes in the future.

If you have any other questions, let me know!

**Reply by Snipppppy** [reply]
wow thank you so much.
That will come in handy
